crcsim/experiment/prepare.py

```diff
@@ -137,6 +140,24 @@ def create_scenarios() -> List:
             .transform(transform_lesion_risk_alpha(1.19))
         )
 
+        scenarios.append(
+            Scenario(name=f"Blood_{scenario}", params=get_default_params())
+            .transform(transform_initial_compliance(screening_rate))
+            .transform(transform_lesion_risk_alpha(1.19))
+        )
+
+        scenarios.append(
+            Scenario(name=f"FIT_Blood_TestCombiningMethod.PARALLEL_{scenario}", params=get_default_params())
+            .transform(transform_initial_compliance(screening_rate))
+            .transform(transform_lesion_risk_alpha(1.19))
+        ) 
+        
+        scenarios.append(
+            Scenario(name=f"FIT_Blood_TestCombiningMethod.SERIAL_{scenario}", params=get_default_params())
+            .transform(transform_initial_compliance(screening_rate))
+            .transform(transform_lesion_risk_alpha(1.19))
+        )
```

Comment:
After we modify the test combining script to set the proportion for combined tests to 0, the base params will assign FIT to all agents. So these tests also need to use `transform_routine_test_proportion` to set the proportion for the relevant test to 1 and all others to 0 (same pattern as Colonoscopy).